### PR TITLE
[freeboxos] First big update of README

### LINKED ISSUE #17074: [freeboxos] Channels are in groups but group is not documented
Channels on a revolution thing are defined in groups.
README provides incomplete channel type IDs without the group.

### LINKED ISSUE #17077: [freeboxos] Trigger channel sysinfo#box-event should be documented
The trigger channel sysinfo#box-event is not documented in README.
Would be interesting to know what events are produced.

### LINKED ISSUE #17080: [freeboxos] 4 channels related to bandwidth not documented
Undocumented channels:
connection-status#bandwidth-up
connection-status#bandwidth-down
connection-status#bandwidth-usage-up
connection-status#bandwidth-usage-down

### PR DESCRIPTION
**Pull request by lolodomo**
Fixes #17074
Fixes #17077
Fixes #17080

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Review by clinique — APPROVED** [review]
I love this ! LGTM

**Review by lsiepel — CHANGES_REQUESTED** [review]
Nice improvement has been done this binding and now also to the documentation! 
One minor comment is that all readme.md headers should be capitalized, otherwise LGTM

**Comment by lolodomo** [comment]
> One minor comment is that all readme.md headers should be capitalized

You mean the first letter ?

**Comment by lolodomo** [comment]
Done

**Comment by lsiepel** [comment]
> > One minor comment is that all readme.md headers should be capitalized
> 
> You mean the first letter ?

Each word.

File: bundles/org.openhab.binding.freeboxos/README.md [hunk 1]
```diff
@@ -45,14 +47,16 @@ Note that the discovered thing will be setup to use only HTTP API (and not HTTPS
...
-
+| Parameter Label | Parameter ID | Description                                                                | Default |
+|-----------------|--------------|----------------------------------------------------------------------------|         |
+| Timeout         | timeout      | The timeout for reading from the API in seconds.                           | 8       |
+| Callback URL    | callbackUrl  | URL to use for playing notification sounds, e.g. 'http://192.168.0.2:8080' |         |
 
 ## Thing Configuration
 
-### API bridge
+### Api bridge
```
Comment (lsiepel) [inline bundles/org.openhab.binding.freeboxos/README.md:57]:
```suggestion
### Api Bridge
```

File: bundles/org.openhab.binding.freeboxos/README.md [hunk 2]
```diff
@@ -67,80 +71,99 @@ FreeboxOS binding has the following configuration parameters:
 If the parameter *apiDomain* is not set, the binding will use the default address used by Free to access your Freebox Server (mafreebox.freebox.fr).
 The bridge thing will initialize only if a valid application token (parameter *appToken*) is filled.
 
-### Server: Revolution or Delta
+### Server things
```
Comment (lsiepel) [inline bundles/org.openhab.binding.freeboxos/README.md:74]:
```suggestion
### Server Thing
```

File: bundles/org.openhab.binding.freeboxos/README.md [hunk 3]
```diff
@@ -67,80 +71,99 @@ FreeboxOS binding has the following configuration parameters:
...
 
 The *revolution* or *delta* thing requires the following configuration parameters:
 
 | Parameter Label  | Parameter ID    | Description                                                              | Required | Default |
 |------------------|-----------------|--------------------------------------------------------------------------|----------|---------|
-| Refresh Interval | refreshInterval | The refresh interval (seconds) which is used to poll the Freebox Server. | No       | 30      |
+| Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll the Freebox Server | No       | 30      |
 
-### Player thing 
+### Player things
```
Comment (lsiepel) [inline bundles/org.openhab.binding.freeboxos/README.md:82]:
```suggestion
### Player Thing
```

File: bundles/org.openhab.binding.freeboxos/README.md [hunk 4]
```diff
@@ -67,80 +71,99 @@ FreeboxOS binding has the following configuration parameters:
...
-| Player port      | port            |                                                                            | No       | 24322   |
 | Password         | password        | AirPlay password                                                           | No       |         |
 | Remote Code      | remoteCode      | Code associated to remote control                                          | No       |         |
 | Accept all MP3   | acceptAllMp3    | Accept any bitrate for MP3 audio or only bitrates greater than 64 kbps     | No       | true    |
-| Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll the player           | Yes      | 30      |
-| Callback URL     | callbackUrl     | URL to use for playing notification sounds, e.g. 'http://192.168.0.2:8080' | No       |         |
+| Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll the Freebox player   | No       | 30      |
 
-### Landline
+### Phone things
```
Comment (lsiepel) [inline bundles/org.openhab.binding.freeboxos/README.md:94]:
```suggestion
### Phone Thing
```

File: bundles/org.openhab.binding.freeboxos/README.md [hunk 5]
```diff
@@ -67,80 +71,99 @@ FreeboxOS binding has the following configuration parameters:
...
-The *landline* thing requires the following configuration parameters:
+The *call* thing requires the following configuration parameters:
 
 | Parameter Label  | Parameter ID    | Description                                                            | Required | Default |
 |------------------|-----------------|------------------------------------------------------------------------|----------|---------|
-| Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll for phone state. | No       | 2       |
+| Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll for phone calls. | No       | 30      |
 
-### Network devices: Host
+### Network device things
```
Comment (lsiepel) [inline bundles/org.openhab.binding.freeboxos/README.md:109]:
```suggestion
### Network Device Thing
```

File: bundles/org.openhab.binding.freeboxos/README.md [hunk 6]
```diff
@@ -67,80 +71,99 @@ FreeboxOS binding has the following configuration parameters:
...
-| mDNS Name        | mDNS            | The mDNS name of the host. Useful in case of virtual MAC.              | No       |         |
+| MAC Address      | macAddress      | The MAC address of the network device.                                 | Yes      |         |
+| Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll the device.      | No       | 30      |
+| mDNS Name        | mDNS            | The mDNS name of the network device. Useful in case of virtual MAC.    | No       |         |
 
 When used, mDNS will search the host based on its mDNS name and eventually update the MAC address accordingly.
 This is useful with devices, especially Apple equipment, that uses randomly generated MAC addresses.
 
-### Repeater and Vm thing 
+### Freeplug thing
```
Comment (lsiepel) [inline bundles/org.openhab.binding.freeboxos/README.md:129]:
```suggestion
### Freeplug Thing
```

File: bundles/org.openhab.binding.freeboxos/README.md [hunk 7]
```diff
@@ -67,80 +71,99 @@ FreeboxOS binding has the following configuration parameters:
...
-|------------------|-----------------|------------------------------------------------------------------|----------|---------|
-| MAC Address      | macAddress      | The MAC address of the player device.                            | No       |         |
-| ID               | id              | Id of the repeater within Freebox Api                            | Yes      | 1       |
-| Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll the player | Yes      | 30      |
+| Parameter Label  | Parameter ID    | Description                                                            | Required | Default |
+|------------------|-----------------|------------------------------------------------------------------------|----------|---------|
+| MAC Address      | macAddress      | The MAC address of the network device.                                 | Yes      |         |
+| Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll the device.      | No       | 30      |
+
+### Repeater thing
```
Comment (lsiepel) [inline bundles/org.openhab.binding.freeboxos/README.md:138]:
```suggestion
### Repeater Thing
```

File: bundles/org.openhab.binding.freeboxos/README.md [hunk 8]
```diff
@@ -67,80 +71,99 @@ FreeboxOS binding has the following configuration parameters:
...
+
+The *repeater* thing requires the following configuration parameters:
+
+| Parameter Label  | Parameter ID    | Description                                                         | Required | Default |
+|------------------|-----------------|---------------------------------------------------------------------|----------|---------|
+
+| ID               | id              | Id of the repeater within Freebox Api                               | Yes      | 1       |
+| Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll the repeater. | No       | 30      |
+
+### Virtual Machine thing
```
Comment (lsiepel) [inline bundles/org.openhab.binding.freeboxos/README.md:148]:
```suggestion
### Virtual Machine Thing
```

File: bundles/org.openhab.binding.freeboxos/README.md [hunk 9]
```diff
@@ -67,80 +71,99 @@ FreeboxOS binding has the following configuration parameters:
...
+
+
+| Parameter Label  | Parameter ID    | Description                                                                | Required | Default |
+|------------------|-----------------|----------------------------------------------------------------------------|----------|---------|
+
+| ID               | id              | Id of the Virtual Machine                                                  | Yes      | 1       |
+| Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll the virtual machine. | No       | 30      |
 
-### Basic shutter thing
+### Home node things
```
Comment (lsiepel) [inline bundles/org.openhab.binding.freeboxos/README.md:159]:
```suggestion
### Home Node Thing
```

**Review by lsiepel — CHANGES_REQUESTED** [review]

**Comment by lolodomo** [comment]
> Each word.

Done.
I kept the plural (Things) when the section concerns several thing types.

**Review by lsiepel — CHANGES_REQUESTED** [review]
Nevermind

**Pull request merged**